src/Listeners/AddWebRoutes.php

```diff
@@ -23,12 +23,12 @@ public function subscribe(Dispatcher $events)
     public function routes(ConfigureForumRoutes $routes)
     {
         $routes->get(
-            '/masquerade/configure',
+            '/profile/configure',
             'masquerade.profile.configure',
             ManageProfileController::class
         );
         $routes->get(
-            '/masquerade/{username}',
+            '/u/{username}/profile',
```

Comment:
I think `/u/username/masquerade` would be better, so you can still use it with other profile extensions